dev/NumberBox/NumberBox.cpp

```diff
@@ -0,0 +1,387 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "NumberBox.h"
+#include "NumberBoxAutomationPeer.h"
+#include "RuntimeProfiler.h"
+#include "ResourceAccessor.h"
+#include "Utils.h"
+
+NumberBox::NumberBox()
+{
+    __RP_Marker_ClassById(RuntimeProfiler::ProfId_NumberBox);
+
+    // Default values for the number formatter
+    auto formatter = winrt::DecimalFormatter();
```

Comment:
>formatter [](start = 9, length = 9)

Is the default decimal formatter shareable? can we make this a static on the class that gets lazily instantiated in case there are a bunch of numberbox's?

Reply:
That is a specific number box's formater though, it would be cool if you could share the unchanged one.

Reply:
If I shared it, that line of code would change all of them, wouldn't it?